turbopack/crates/turbopack-core/src/chunk/chunking_context.rs

```diff
@@ -133,13 +133,22 @@ pub struct ChunkingConfig {
 #[turbo_tasks::value(transparent)]
 pub struct ChunkingConfigs(FxHashMap<ResolvedVc<Box<dyn ChunkType>>, ChunkingConfig>);
 
+#[turbo_tasks::value(shared)]
+#[derive(Debug, Clone, Copy, Hash, TaskInput, Default)]
+pub enum SourceMapSourceType {
+    AbsoluteFileUri,
+    RelativeFileUri,
```

Comment:
Technically it's not a FileUri, it's just a relative uri.


```suggestion
    Relative,
```